src/stores/game.js

```diff
@@ -13,22 +13,21 @@ function queenCount(player) {
   }
   return player.faceCards.reduce((queenCount, card) => queenCount + (card.rank === 12 ? 1 : 0), 0);
 }
-  
+
 const compareByRankThenSuit = (card1, card2) => {
-  return (card1.rank - card2.rank) || (card1.suit - card2.suit);
+  return card1.rank - card2.rank || card1.suit - card2.suit;
 };
 
 const setPlayers = (player, myPnum) => {
-  const sortP1 = (cards) => player.pNum === myPnum ? cards?.sort(compareByRankThenSuit) : cards;
+  const sortP1 = (cards) => (player.pNum === myPnum ? cards?.sort(compareByRankThenSuit) : cards);
   return {
     ...player,
-    hand: sortP1(player.hand)?.map((card) => createGameCard(card)),
-    points: sortP1(player.points)?.map((card) => createGameCard(card)),
-    faceCards: sortP1(player.faceCards)?.map((card) => createGameCard(card))
+    hand: sortP1(player.hand)?.map((card) => createGameCard(card)) ?? [],
+    points: sortP1(player.points)?.map((card) => createGameCard(card)) ?? [],
+    faceCards: sortP1(player.faceCards)?.map((card) => createGameCard(card)) ?? [],
```

Comment:
TODO: see if the fallbacks are necessary now that we are populating the full game after rematch